### ISSUE TITLE
Emphasize vs. Italic

### ISSUE THREAD

**Issue by kloetzl** [katrinleinweber/Markdown-for-bio-scientists#1]
I fear I am being nit-picky here, but `\emph{Species name}` is not the same as `<i>Species name</i>`. As the name `emph` indicates, it is used to emphasize things. The effect can be bold, italic or underlined (via the `ulem` package). Thus the equivalent in HTML is `<em>`.

The difference is that emphasize is a semantic choice, whereas italic is stylistic and thus inherently cannot be expressed via semantic markup. Which is also true for Markdown, unless you see Markdown as a subset of HTML which allows the use of `<i>`. So technically, there is no way to correctly make a species name italic in plain markdown.

Summing up: `\emph{Important}` ↔ `<em>Important</em>` ↔`*Important*`
`\textit{E. coli}` ↔ `<i>E. Coli</i>`

**Comment by katrinleinweber** [comment]
True. Maybe emphasising a species name is itself wrong, but that is [pandoc's](https://github.com/jgm/pandoc/blob/bf7d858a9e1ec788bf80c489186ee5f8b7d4bcdd/src/Text/Pandoc/Readers/LaTeX.hs#L431) [doing](https://github.com/jgm/pandoc/issues/1142).

**Comment by kloetzl** [comment]
> …, but that is pandoc's doing.

Meh, tools trying to be clever. m(

**Comment by katrinleinweber** [comment]
Hm, it's defined that way in [Markdown itself](https://daringfireball.net/projects/markdown/basics), so it's probably me using it incorrectly for species names.

**Comment by kloetzl** [comment]
The problem with species names is that you are trying to change its fontface, which should not be possible with semantic mark up. Unfortunately there is no way to express _this is a species name_ in Markdown. ☺ In Latex one would simply create a new command.

**Comment by katrinleinweber** [comment]
[Already done](https://tex.stackexchange.com/questions/71092/), but this talk was trying to be beginner friendly ;-)
